drivers/at25xxx/at25xxx.c

```diff
@@ -81,12 +81,12 @@ static inline int _wait_until_eeprom_ready(const at25xxx_t *dev)
     return tries == 0 ? -ETIMEDOUT : 0;
 }
 
-static ssize_t _write_page(const at25xxx_t *dev, uint32_t pos, const void *data, size_t len)
+int at25xxx_write_page(const at25xxx_t *dev, uint32_t page, uint32_t offset, const void *data, size_t len)
```

Comment:
`at25xxx_write_page()` does not acquire the SPI bus but `at24cxxx_write_page()` does acquire the I2C bus.

Reply:
Good catch!